tfhe/src/integer/server_key/comparator.rs

```diff
@@ -2115,210 +2381,372 @@ mod tests {
     }
 
     //=============================================================
-    // Scalar comparison tests
+    // Signed comparison tests
     //=============================================================
 
-    /// Function to test an "unchecked_scalar" compartor function.
+    /// Function to test an "unchecked" compartor function.
     ///
-    /// This calls the `unchecked_scalar_comparator_method` with fresh ciphertexts
+    /// This calls the `unchecked_comparator_method` with fresh ciphertexts
     /// and compares that it gives the same results as the `clear_fn`.
-    fn test_unchecked_scalar_function<UncheckedFn, ClearF>(
+    fn test_signed_unchecked_function<UncheckedFn, ClearF>(
         param: ClassicPBSParameters,
         num_test: usize,
         unchecked_comparator_method: UncheckedFn,
         clear_fn: ClearF,
     ) where
-        UncheckedFn:
-            for<'a, 'b> Fn(&'a Comparator<'b>, &'a RadixCiphertext, U256) -> RadixCiphertext,
-        ClearF: Fn(U256, U256) -> U256,
+        UncheckedFn: for<'a, 'b> Fn(
+            &'a Comparator<'b>,
+            &'a SignedRadixCiphertext,
+            &'a SignedRadixCiphertext,
+        ) -> SignedRadixCiphertext,
+        ClearF: Fn(i128, i128) -> i128,
     {
         let mut rng = rand::thread_rng();
 
-        let num_block = (256f64 / (param.message_modulus.0 as f64).log(2.0)).ceil() as usize;
+        let num_block = (128f64 / (param.message_modulus.0.ilog2() as f64)).ceil() as usize;
```

Comment:
we do 128 bits comps now vs 256 before ?

Reply:
what do you mean ? we are testing smaller things now right while we now have access to I256 ? not sure I understand 🤔

Reply:
- unsigned radix comparisons tests are still on U256
- signed radix comparisons tests are on i128 because when written I256 was not available
- I did not upgrade these tests to I256 because if It works on i128, it works i256

Reply:
my mistake, I misread